storage/storagepb/kv.proto

```diff
@@ -14,11 +14,14 @@ message KeyValue {
   int64 create_revision = 2;
   // mod_revision is the last modified revision of the key.
   int64 mod_revision = 3;
+  // tombstone_revision is the tombstone revision of the key.
+  // If the key is not tombstoned, its revision is set to zero.
+  int64 tombstone_revision = 4;
```

Comment:
Does it mean that we will also return the tombstone_rev back to the user? Why do we want to do that? We have mod_rev since it will be returned to user.